odysseus/halow_setup/setup_system.sh

```diff
@@ -50,8 +50,8 @@ mv /etc/wpa_supplicant/wpa_supplicant.conf /etc/wpa_supplicant/wpa_supplicant.co
 
 
 echo "7. Enabling i2c, mac80211"
-output_i2c="$(cat "/etc/modules" | grep -e "i2c-dev")"
-output_mac="$(cat "/etc/modules" | grep -e "mac80211")"
+output_i2c="$(grep -e "i2c-dev" < "/etc/modules")"
```

Comment:
uh wait why do we need to find an I2C device driver? Not bad just curious

Reply:
I was wondering the same thing I put this in the README: `TODO: Does nrc actually use i2c?`
I tried Halow without it and it worked fine, I probably originally was basing my decision on this: https://docs.alfa.com.tw/Product/AHPI7292S/30_Technical_Details/
The i2c interface does relay some data to the pi, but that data is uneeded and it not userspace, so defintely not needed.

I removed that code.